src/amber/router/request.cr

```diff
@@ -0,0 +1,55 @@
+require "./params"
+require "./parser"
+require "./file"
+
+class HTTP::Request
+  include Amber::Router::Parser
```

Comment:
Why include all of this into request when we could just call `Amber::Router::Parser.params` as a class method and have it generate params. Which we could save here in the memo @params. That way call chains are partitioned, variable definitions stay within scope and code follows SOLID principles.

Reply:
I had it that way :)